src/libZilliqa/Zilliqa.cpp

```diff
@@ -77,9 +92,22 @@ void Zilliqa::ProcessMessage(pair<vector<unsigned char>, Peer>* message) {
         return;
       }
 
+      const auto ins_byte = message->first.at(MessageOffset::INST);
```

Comment:
Should we add some kind of `ifdef` or check for a constant value around the newly added code here, so that we don't always include benchmarking in our binary?

Reply:
Maybe we can only print out bench mark information when debug level is higher, normally we don't log?